tests/unit/lms/data_tasks/report/create_from_scratch/test_01_functions/test_01_date_functions.py

```diff
@@ -9,29 +9,17 @@
 
 
 class TestDateFunctions:
-    NOW = datetime(2025, 3, 6, tzinfo=UTC)  # We hardcode the date to avoid flakiness
-    # 2024 was a leap year so the math accounts for that 366 days
-    ONE_YEAR_AGO = (NOW - timedelta(days=366)).date()
-    TWO_YEARS_AGO = (NOW - timedelta(days=366 + 365)).date()
+    # We hardcode the date to avoid flakiness
+    NOW = date(2025, 3, 6)
```

Comment:
Use date directly instead of creating a tz aware datetime and converting to date.